birdhouse/config/thredds/catalog.xml.template

```diff
@@ -31,7 +31,7 @@
 
     </datasetScan>
 
-    <datasetScan name="Datasets" ID="datasets" path="datasets" location="/pavics-ncml">
+    <datasetScan name="${STAC_DATA_LOCATION_NAME}" ID="${STAC_DATA_LOCATION_NAME_lower}" path="${STAC_DATA_LOCATION_NAME_lower}" location="${STAC_DATA_LOCATION_FOR_THREDDS_ON_CONTAINER}">
```

Comment:
This variable name assumes `components/stac` will be present, but nothing guarantees nor even require this component to be loaded. Avoid mixing `STAC_` prefixed variables that should be reserved only to this component.

However, this part of the XML could be injected entirely using an environment variable (or an extra XML if THREDDS allows load others?), when STAC+THREDDS are enabled.  That would be an XML defined somewhere in `components/stac/config/thredds`.

Reply:
Exactly. Thanks for providing the example I had in mind.

Reply:
> This variable name assumes `components/stac` will be present, but nothing guarantees nor even require this component to be loaded.

That's a good point and the solutions suggested could work. If I recall there is a current consensus (or emerging consensus?) that STAC should be a required component of a minimum DACCS (Marble?) node even if no data is served at the node. If that were to be the case, then would the existing structure suffice?